DOM - It's now possible to define history settings for DomInstances on DomDefinition level (#2457)
SkylineCommunications/dataminer-docs: user-guide/Advanced_Modules/DOM/DOM_ModuleSettings/DOM_DomInstanceHistorySettings.md

# DomInstanceHistorySettings

>[!NOTE]

> > From 10.4.3/10.5.0 onwards<!-- RN38294 -->, it is possible to override this setting in a `DomDefinition` with the [ModuleSettingsOverrides](xref:DomDefinition#modulesettingsoverrides) property.

Author: ```suggestion
> From 10.4.3/10.5.0 onwards<!-- RN38294 -->, it is possible to override this setting on `DomDefinition` level with the [ModuleSettingsOverrides](xref:DomDefinition#modulesettingsoverrides) property.
```
